src/Symfony/Component/DependencyInjection/Tests/Compiler/AnalyzeServiceReferencesPassTest.php

```diff
@@ -81,7 +81,7 @@ public function testProcessMarksEdgesLazyWhenReferencedServiceIsLazy()
         $this->assertCount(1, $graph->getNode('b')->getInEdges());
         $this->assertCount(1, $edges = $graph->getNode('a')->getInEdges());
 
-        $this->assertSame($ref2, $edges[0]->getValue());
+        $this->assertEquals($ref2, $edges[0]->getValue());
```

Comment:
Why changing these assertions ?

Reply:
Indeed, `assertSame` is more reliable BTW.